docs/README.md

```diff
@@ -0,0 +1,171 @@
+# Shortest
+
+[![npm version](https://badge.fury.io/js/shortest.svg)](https://badge.fury.io/js/shortest)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](http://makeapullrequest.com)
```

Comment:
Not a fan of anything that isn't essential, images that need to be loaded, etc. license is in another file already